diamond.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
```

Comment:
close! you got half the diamond done correctly. your code did have 2 warnings - watch out for those!